Drop-In/src/main/java/com/braintreepayments/api/dropin/DropInActivity.java

```diff
@@ -280,6 +282,14 @@ public void onPaymentMethodNoncesUpdated(List<PaymentMethodNonce> paymentMethodN
             if (mDropInRequest.isVaultManagerEnabled()) {
                 mVaultManagerButton.setVisibility(View.VISIBLE);
             }
+
+            for (PaymentMethodNonce nonce : paymentMethodNonces) {
+                if (nonce.getClass() == CardNonce.class) {
+                    mBraintreeFragment.sendAnalyticsEvent("vaulted-card.appear");
+                    break;
```

Comment:
There may be more than 1 vaulted card

Reply:
We want to only send this analytic event once. One time for when vaulted card (or multiple) appear on the screen.

Reply:
Ok that makes sense!